.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This generated file doesn't really belong here. 
Could you delete it, and and add it to `.gitignore`